docs/developer/chart-repo.md

```diff
@@ -0,0 +1,62 @@
+# Kubeapps chart-repo Developer Guide
+
+The `chart-repo` component is tool that scans a Helm chart repository and populates chart metadata in a MongoDB server. This metadata is then served by the `chartsvc` component. Its source is maintained in the [Monocular project repository](https://github.com/helm/monocular).
+
+## Prerequisites
+
+- [Git](https://git-scm.com/)
+- [Make](https://www.gnu.org/software/make/)
+- [Go programming language](https://golang.org/dl/)
+- [Docker CE](https://www.docker.com/community-edition)
+- [Kubernetes cluster (v1.8+)](https://kubernetes.io/docs/setup/pick-right-solution/). [Minikube](https://github.com/kubernetes/minikbue) is recommended.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Telepresence](https://telepresence.io)
+
+## Environment
+
+```bash
+export GOPATH=~/gopath/
+export PATH=$GOPATH/bin:$PATH
+export MONOCULAR_DIR=$GOPATH
+```
+
+## Download the Monocular source code
+
+```bash
+git clone https://github.com/helm/monocular $MONOCULAR_DIR
+```
+
+The `chart-repo` sources are located under the `cmd/chart-repo/` directory.
+
+### Install Kubeapps in your cluster
+
+Kubeapps is a Kubernetes-native application. To develop and test Kubeapps components we need a Kubernetes cluster with Kubeapps already installed. Follow the [Kubeapps installation guide](../../chart/kubeapps/README.md) to install Kubeapps in your cluster.
+
+### Building the `chart-repo` image
+
+```bash
+cd $MONOCULAR_DIR
+dep ensure
+make -C cmd/chart-repo docker-build
+```
+
+This builds the `chart-repo` Docker image.
+
+### Running in development
```

Comment:
shouldn't we just point here to the monocular repo? We are having duplicated documentation

Also this is likely to get outdated since the source code of this tool is not in this repo.

Reply:
See https://github.com/kubeapps/kubeapps/pull/775#issuecomment-434343709.

> Also this is likely to get outdated since the source code of this tool is not in this repo.

 But how often do we need to change the development workflow?

Reply:
I know that we don't update them very often